client/src/app/modules/events/event-edit/event-edit.component.scss

```diff
@@ -0,0 +1,22 @@
+*::-webkit-input-placeholder {
```

Comment:
These changes were just copied from web, the placeholder style (color & font style) was too much the same as the styling of real input data, so I added different color and italic style for placeholders

Reply:
To be honest, I dont want our app to support IE :), unless we change our minds

Reply:
As I said, I copied it... I can remove the IE part no problem ;)